src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResource.java

```diff
@@ -404,7 +404,7 @@ static String getResource(String key) {
                     "FEDAUTHINFO token stream is not long enough ({0}) to contain the data it claims to."},
             {"R_FedAuthInfoDoesNotContainStsurlAndSpn",
                     "FEDAUTHINFO token stream does not contain both STSURL and SPN."},
-            {"R_ADALExecution", "Failed to authenticate the user {0} in Active Directory (Authentication={1})."},
+            {"R_MSALExecution", "Failed to authenticate the user {0} in Active Directory (Authentication={1})."},
```

Comment:
Please search for `ADAL` in SQLServerResource and make similar changes to the error messages that mentioned ADAL.